File: ct/ct.go
package main

import (
	"fmt"
	"time"
)

type SpkerInfoTmp struct {
	NVectordim int32
	PfIvector [512]float32
	NSpkId   int32
	NSpkNum  int32
	NBlockNum int32
	Ntb int32
	SpkName   [128]byte
	BUpdate  byte
}

func febo(n int) int {
	f:=make([]int,101)
	f[0]=0
	f[1] =1;
	for i:=2;i<=n;i++{
		f[i] = f[i-1]+f[i-2]
	}
	return f[n]
}

func main(){
	st:=time.Now()
	for i:=0;i<1000000;i++{
		febo(100)
	}
	fmt.Println(time.Since(st))
}

File: ct/ct.c

#include<stdio.h>
#define bool char
#define true 1
#define false 0
typedef struct Info{
    int vm;
    float pfi[512];
    int nspkId;
    int nspknum;
    int nb;
    int ntb ;
    char spkname[128];
    bool update;

}INFO;


int main(){
    printf("%d",sizeof(INFO));
    printf("\n%d,%d",true,false);
}